### PR TITLE
Keypair: tweak Render logic

### PR DESCRIPTION
**Pull request by justinsb**
We don't need to special-case it

### CODE REVIEW

File: upup/pkg/fi/fitasks/keypair.go [hunk 1]
```diff
@@ -190,9 +191,9 @@ func (_ *Keypair) Render(c *fi.Context, a, e, changes *Keypair) error {
 		} else if changes.Type != "" {
 			createCertificate = true
 			glog.V(8).Infof("creating certificate new Type")
-		} else if a.Format != "" {
-			// We only want to log that we are ignoring the changes if we are not going to save a new
-			// keypair.yaml file.  If a.Format is empty we are going to save a new keypar.yaml file.
+		} else if changes.Format != "" {
+			changeStoredFormat = true
```
Comment (chrislovecnm) [inline upup/pkg/fi/fitasks/keypair.go]: Can we move the log message that you deleted to here

`glog.Infof("updated Legacy Keypair for: %q, to newer Keypair API format", name)`

Comment (justinsb) [reply upup/pkg/fi/fitasks/keypair.go]: Good call - I'll restore the message.  We don't want to log "updated" until we've done it though :-)

**Comment by justinsb** [comment]
Message updated!

**Pull request merged**